code/datums/namepool.dm

```diff
@@ -35,3 +35,9 @@ GLOBAL_LIST_EMPTY_TYPED(namepool, /datum/namepool)
 	firstname_male_pool = "names/moth_first"
 	firstname_female_pool = "names/moth_first"
 	lastname_pool = "names/moth_last"
+
+/datum/namepool/sectoid
+	lastname_pool = "names/sectoid_last"
+
```

Comment:
```suggestion
```

Reply:
This isn't used anymore correct?

Reply:
yeah my bad, i kinda forgot what i needed to remove